### ISSUE TITLE
V3 MAJOR BREAKING CHANGES and change in direction.

### ISSUE THREAD

**Issue by THEGOLDENPRO** [THEGOLDENPRO/devgoldyutils#3]
TLDR: v2 was full of junk of that **shouldn't** be used so I removed them all. Sadly this leads to breakage of some projects.

As the tldr states, v3 will break some projects, one of them being the [goldy bot framework](https://github.com/Goldy-Bot/Goldy-Bot-Framework) (however, this will be fixed in the new framework). For any legacy projects hard set your version to [2.5.8](https://pypi.org/project/devgoldyutils/2.5.8/) ( ``devgoldyutils==2.5.8`` ), for everything else I would recommend to drop using the utils removed and implement them yourself.

The devgoldyutils library will now only service the following utils:
- [Logging](https://github.com/THEGOLDENPRO/devgoldyutils/blob/v3/devgoldyutils/logging.py) (logger adaptor and custom logger format)
- [Colours](https://github.com/THEGOLDENPRO/devgoldyutils/blob/v3/devgoldyutils/colours.py) (console colours)
- [Strings](https://github.com/THEGOLDENPRO/devgoldyutils/blob/v3/devgoldyutils/strings.py) (string and path shortener)
- [Debugging](https://github.com/THEGOLDENPRO/devgoldyutils/blob/v3/devgoldyutils/debug.py) (quick print debug util)
